script/package.ts

```diff
@@ -52,7 +53,7 @@ function packageWindows() {
     'cleanup-windows-certificate.ps1'
   )
 
-  if (isAppveyor() || isGitHubActions()) {
+  if (isGitHubActions()) {
```

Comment:
Should we check if isPublishable too?

```suggestion
  if (isGitHubActions() && isPublishable()) {
```